src/main/java/ch/kalunight/zoe/service/InfoPanelRefresher.java

```diff
@@ -630,6 +634,163 @@ private String refreshPannel() throws SQLException {
     return stringMessage.toString();
   }
 
+  private void pseudoList(final StringBuilder stringMessage, List<DTO.Player> playersList) throws SQLException {
+    for(DTO.Player player : playersList) {
+
+      List<DTO.LeagueAccount> leagueAccounts = player.leagueAccounts;
+
+      List<DTO.LeagueAccount> accountsInGame = new ArrayList<>();
+      List<DTO.LeagueAccount> accountNotInGame = new ArrayList<>();
+
+      for(DTO.LeagueAccount leagueAccount : leagueAccounts) {
+        DTO.CurrentGameInfo currentGameInfo = CurrentGameInfoRepository.getCurrentGameWithLeagueAccountID(leagueAccount.leagueAccount_id);
+        if(currentGameInfo != null) {
+          accountsInGame.add(leagueAccount);
+        }else {
+          accountNotInGame.add(leagueAccount);
+        }
+      }
+
+      if(accountsInGame.isEmpty()) {
+
+        if(ConfigRepository.getServerConfiguration(guild.getIdLong()).getInfopanelRankedOption().isOptionActivated()) {
+
+          if(accountNotInGame.size() == 1) {
+
+            LeagueAccount leagueAccount = accountNotInGame.get(0);
+
+            getTextInformationPanelRankOption(stringMessage, player, leagueAccount, false);
+          }else {
+            
+            stringMessage.append(String.format(LanguageManager.getText(server.serv_language, "infoPanelRankedTitleMultipleAccount"), player.user.getAsMention()) + "\n");
+            
+            for(DTO.LeagueAccount leagueAccount : accountNotInGame) {
+              
+              getTextInformationPanelRankOption(stringMessage, player, leagueAccount, true);
+            }
+          }
+
+        } else {
+          notInGameWithoutRankInfo(stringMessage, player);
+        }
+      }else if (accountsInGame.size() == 1) {
+        stringMessage.append(player.user.getAsMention() + " : " 
+            + InfoPanelRefresherUtil.getCurrentGameInfoStringForOneAccount(accountsInGame.get(0), server.serv_language) + "\n");
+      }else {
+        stringMessage.append(player.user.getAsMention() + " : " 
+            + LanguageManager.getText(server.serv_language, "informationPanelMultipleAccountInGame") + "\n"
+            + InfoPanelRefresherUtil.getCurrentGameInfoStringForMultipleAccounts(accountsInGame, server.serv_language));
+      }
+    }
+  }
+
+  private void getTextInformationPanelRankOption(final StringBuilder stringMessage, DTO.Player player,

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'getTextInformationPanelRankOption(StringBuilder, DTO.Player, DTO.LeagueAccount, boolean)' has an NPath complexity of 2736](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=5406891)